t/ui/14-dashboard-parents.t

```diff
@@ -146,5 +146,28 @@ subtest 'filtering subgroups' => sub {
     isnt(scalar @{$driver->find_elements('opensuse test', 'link_text')}, 0, "child group 'opensuse test' present'");
 };
 
+subtest 'View grouped by group' => sub {
+    my $parent_group_id = $schema->resultset('JobGroupParents')->search(undef, {name => 'Test parent'})->first->id;
+    $driver->get('/parent_group_overview/'. $parent_group_id);
+    wait_for_ajax_and_animations;
```

Comment:
please add a message explaining the step, e.g. `wait_for_ajax_and_animations(msg => 'waiting before clicking "grouped_by_group"')`